### PR TITLE
Make MTO Available to prime

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Make the MTO available to the prime

## Reviewer Notes
- Currently, it is always making the same MTO available to the prime. We can make it a per move_order basis once we have those pieces in place

## Setup
`make server_run`
`make server_test`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References
* [Jira story](https://dp3.atlassian.net/browse/MB-594) for this change

### CODE REVIEW

File: pkg/testdatagen/shared.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ type Assertions struct {
 	ElectronicOrdersRevision                 models.ElectronicOrdersRevision
 	Entitlement                              models.Entitlement
 	FuelEIADieselPrice                       models.FuelEIADieselPrice
+	GHCEntitlement                           models.GHCEntitlement
```
Comment (ralren) [inline pkg/testdatagen/shared.go]: Please use `Entitlement` instead.

File: pkg/models/move_task_order.go [hunk 2]
```diff
@@ -15,14 +15,24 @@ import (
...
+	ReferenceID              *string        `db:"reference_id"`
+	IsAvailableToPrime       bool           `db:"is_available_to_prime"`
+	IsCanceled               bool           `db:"is_canceled"`
+	CreatedAt                time.Time      `db:"created_at"`
+	UpdatedAt                time.Time      `db:"updated_at"`
+	DestinationAddress       Address        `belongs_to:"addresses"`
+	DestinationAddressID     uuid.UUID      `db:"destination_address_id"`
+	DestinationDutyStation   DutyStation    `belongs_to:"duty_stations"`
+	DestinationDutyStationID uuid.UUID      `db:"destination_duty_station_id"`
+	Entitlements             GHCEntitlement `has_one:"entitlements"`
```
Comment (ralren) [inline pkg/models/move_task_order.go]: `Entitlements` come from `MoveOrders` now

File: migrations/20191216155742_add-back-ghc-entitlements.up.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE ghc_entitlements
```
Comment (ralren) [inline migrations/20191216155742_add-back-ghc-entitlements.up.sql]: There is an existing `entitlements` table. Please delete.

File: pkg/models/move_task_order.go [hunk 4]
```diff
@@ -15,14 +15,23 @@ import (
...
+	ID                       uuid.UUID   `db:"id"`
+	MoveOrder                MoveOrder   `belongs_to:"move_orders"`
+	MoveOrderID              uuid.UUID   `db:"move_order_id"`
+	ReferenceID              *string     `db:"reference_id"`
+	IsAvailableToPrime       bool        `db:"is_available_to_prime"`
+	IsCanceled               bool        `db:"is_canceled"`
+	CreatedAt                time.Time   `db:"created_at"`
+	UpdatedAt                time.Time   `db:"updated_at"`
+	DestinationAddress       Address     `belongs_to:"addresses"`
+	DestinationAddressID     uuid.UUID   `db:"destination_address_id"`
```
[the remark this answers is not in the document]
Comment (Ronolibert) [reply pkg/models/move_task_order.go]: No reason, didn't know it was already there. Good catch!

File: pkg/models/move_task_order.go [hunk 5]
```diff
@@ -15,14 +15,23 @@ import (
...
+	DestinationDutyStation   DutyStation `belongs_to:"duty_stations"`
```
[the remark this answers is not in the document]
Comment (Ronolibert) [reply pkg/models/move_task_order.go]: Yup! I think all these were artifacts from our old implementation. Removed the duplicate fields

**Pull request merged**